### PR TITLE
Publish Play and Tactic info

### PR DESCRIPTION
**Pull request by MathewMacDougall**
### Description

The AI now published information about the current Play and Tactics that are running. The new topic is `backend/play_info`.

Example output from rostopic:
```
play_type: "Unknown"
play_name: "Example Play"
tactics: [Robot 0  -  Move Tactic, Robot 1  -  Move Tactic, Robot 2  -  Move Tactic, Robot 3  -  Move Tactic, Robot 5  -  Move Tactic, Robot 6  -  Move Tactic]
```

I'm leaving the play type as "unkown" for now until we handle refbox states better.

### Testing Done

Verified manually messages are constructed and published correctly.

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in the `thunderbots/software/geom` folder for examples.
- [x] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `thunderbots/software/geom`
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/thunderbots_msgs/msg/PlayInfo.msg
Comment (garethellis0) [inline src/thunderbots_msgs/msg/PlayInfo.msg]: could we change to `robot_tactic_assignment`?

**Review by garethellis0 — COMMENTED** [review]
LGTM, just need to rebase to hopefully clear up all the receiver changes that are showing up?

File: src/thunderbots_msgs/msg/PlayInfo.msg
Comment (MathewMacDougall) [reply src/thunderbots_msgs/msg/PlayInfo.msg]: renamed

**Comment by MathewMacDougall** [comment]
My bad, didn't realize I had the receiver changes in here too. They're removed now.

**Review by garethellis0 — APPROVED** [review]
LGTM :+1: :tada:

**Comment by MathewMacDougall** [comment]
Going to wait for approval from @qgolsteyn and/or @kelvinkoon 	before merging.

File: src/thunderbots_msgs/msg/PlayInfo.msg
Comment (qgolsteyn) [inline src/thunderbots_msgs/msg/PlayInfo.msg:8]: Based on implementation, I should expect a string here of format `"Robot X - Tactic A"`?

Does the string at index 0 correspond to the tactic for robot 0 and so on?

**Review by qgolsteyn — APPROVED** [review]
LGTM :+1:

File: src/thunderbots_msgs/msg/PlayInfo.msg
Comment (qgolsteyn) [reply src/thunderbots_msgs/msg/PlayInfo.msg:8]: Not blocking, just asking for clarification

Comment (MathewMacDougall) [reply src/thunderbots_msgs/msg/PlayInfo.msg:8]: Yes, strings will be in the form `"Robot X - Tactic A"`.

The strings are sorted in order of ascending robot id. So it may be something like:
```
[
    "Robot 0 - Move Tactic",
    "Robot 1 - Move Tactic",
    "Robot 3 - Move Tactic",
    "Robot 4 - Move Tactic",
    "Robot 7 - Move Tactic",
]
```

We don't send a string for robots without a tactic (eg. the robot isn't even on the field).

**Pull request merged**
